usaspending_api/common/helpers/sql_helpers.py

```diff
@@ -170,68 +166,43 @@ def convert_composable_query_to_string(sql, model=Award, cursor=None):
     return sql
 
 
-def execute_update_sql(sql, model=Award):
-    """
-    Executes a sql query against a database that perform some sort of update
-    (INSERT, UPDATE, etc).
-
-    sql     - Can be either a sql string statement or a psycopg2 Composable
-              object (Identifier, Literal, SQL, etc).
-    model   - A Django model that represents a database table germaine to your
-              query.  If one is not supplied, Award will be used since it is
-              fairly central to the database as a whole.
-
-    Returns a CLOSED cursor.  Can be used for row counts and what not, but is
-    no longer operational.
-    """
-    connection = get_connection(model, False)
-    with connection.cursor() as cursor:
-        # Because django-debug-toolbar does not understand Composable queries,
-        # we need to convert the query to a string before executing it.
-        cursor.execute(convert_composable_query_to_string(sql, cursor=cursor))
-        return cursor
+def cursor_fetcher(cursor):
+    """ Fetcher that simply returns the cursor. """
+    return cursor
 
 
-def execute_fetchall(sql, model=Award, fetcher=None):
+def default_fetcher(cursor):
```

Comment:
I preferred the original func name, I understand what "fetchall" is in context of a DB cursor. `default_fetcher()` doesn't convey much and _requires_ the docstring be read, which I argue is a code smell.

perhaps `generic_fetchall` or `default_fetchall` or `retreive_db_records` or ...